docs/getting-started/delta-lake-format.md

```diff
@@ -115,22 +116,15 @@ We will now start Polynote in a docker container, and an external Metastore (Der
 To do so, run the following commands in the projects root directory:
     
     docker-compose build
-    mkdir -p data/_metastore
+    mkdir -f data
+    mkdir -f data/_metastore
```

Comment:
Same as above. If you meant `-p`, the first command can be removed

Reply:
actually you right: -p only fails when data and data/_metastore exist, which makes sense. I will keep this one with -p